packages/block-library/src/column/deprecated.js

```diff
@@ -6,9 +6,34 @@ import classnames from 'classnames';
 /**
  * WordPress dependencies
  */
-import { InnerBlocks } from '@wordpress/block-editor';
+import { InnerBlocks, useBlockProps } from '@wordpress/block-editor';
 
 const deprecated = [
+	{
```

Comment:
I'm not really sure about the deprecation here. For start it should have `attributes` or `isEligible` etc.. With just a `save` function, wouldn't it match if a Columns block was invalid for whatever reason? -cc @gziolo

Reply:
It would work without `attributes`, but as soon as they change, it might stop working. So it's safer to set them. `isEligible` is optional.